ui/app/AppMain.qml

```diff
@@ -115,7 +115,6 @@ RowLayout {
         property bool displayChatImages: false
         property bool useCompactMode: true
         property bool timelineEnabled: true
-        property string locale: "en"
```

Comment:
One thing to note here, the ne default of `globalSettings` will be `en` as well. However, if an existing user has already changed the locale to something else, this value will not be used, but whatever is stored in the settings file (which values are exposed under `appSettings`).

Since we're using `globalSettings.locale` from this commit on, the language will be reset to `en` the first time a user uses a version with this change (because it's now read from the global settings file which also defaults to `en`)

I thought about introducing a routine that checks whether `appSettings.locale` exists and doesn't match `globalSettings.locale` and would then set `globalSettings.locale` to that value.

However, to do this only once we need to ensure `appSettings.locale` is erased after that, which we can't do cause it's declaratively defined in QML.

So I went with just removing it all together and have users be a little bit surprised instead.